src/clj_puppetdb/vcr.clj

```diff
@@ -22,21 +21,29 @@
         (assoc response :content-type
                         {:mime-type (.getMimeType content-type)
                          :charset   (.getCharset content-type)})))))
+
 (defn body->string
   [response]
   "Turn response body into a string."
   (let [charset (util/response-charset response)]
-    (update-in response [:body] #(slurp (BufferedReader. (InputStreamReader. % charset))))))
+    (->> (-> (get response :body)
```

Comment:
I'm afraid we have to respect the encoding in the message header here, otherwise we might be misinterpreting the data.
But we can drop similar construct in the vcr.clj where I quite arbitrarily introduced the:
```clojure
(def vcr-charset (Charset/forName "UTF-8"))
```
I'll drop that.